src/vendor/wordpress/jetpack-blocks.stories.mdx

```diff
@@ -0,0 +1,29 @@
+import { Story, Canvas, Meta } from '@storybook/addon-docs/blocks';
+
+<Meta title="Vendor/WordPress/Jetpack Blocks" />
+
+# Jetpack Blocks
+
+WordPress developer Automattic provides a [Jetpack](https://jetpack.com/) plugin that includes additional blocks. We support a selection of these.
+
+## Markdown Block
+
+Reenforces the containing element's rhythm. If contained within a [Rhythm object](/docs/objects-rhythm--default-story), its children will inherit the same spacing.
```

Comment:
>If contained within a [Rhythm object](/docs/objects-rhythm--default-story), its children will inherit the same spacing.

I'm a little confused by this. It appears that the CSS below will apply the rhythm mixin regardless of whether the markdown block is in a rhythm object or not? Am I missing something?

Reply:
Oh, do you mean that the level of spacing will be inherited via custom props? (e.g. if the parent has generous spacing then the markdown block will also have generous spacing?) That's pretty clever! I wonder if this comment could be rewritten somehow to make that clearer?

Reply:
```suggestion
The markdown block applies vertical rhythm between its children. If contained within a [Rhythm object](/docs/objects-rhythm--default-story), the markdown block will inherit the same spacing. For example, if the Rhythm object uses the `rhythm--generous` modifier, the markdown block will inherit the generous spacing.
```

Or something...